renderer/BUILD.gn

```diff
@@ -23,6 +23,7 @@ source_set("renderer") {
     "//brave/components/brave_shields/common",
     "//brave/components/brave_wallet/renderer",
     "//brave/components/cosmetic_filters/renderer",
+    "//brave/components/script_injector/renderer",
```

Comment:
this is not needed because the dependency is only referenced in brave_content_renderer_client.cc so https://github.com/brave/brave-core/pull/20336/files#diff-a82c29dd89e671c74dee5f53c73bb157367fa270cfaa02f80f076490bd257ba3R34 is sufficient